src/Windowing/Silk.NET.Windowing.Common/GraphicsAPI.cs

```diff
@@ -0,0 +1,40 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
+using System;
+
+namespace Silk.NET.Windowing.Common
+{
+    /// <summary>
+    /// Represents the context API, and associated configuration, that the window should use.
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    /// By default, this struct uses OpenGL 3.3, core profile, forward-compatable.
+    /// </para>
+    /// </remarks>
+    public struct GraphicsAPI
+    {
+        /// <summary>
+        /// The rendering API to use.
+        /// </summary>
+        public ContextAPI API { get; set; }
+
+        /// <summary>
+        /// The profile to use.
+        /// </summary>
+        public ContextProfile Profile { get; set; }
+
+        /// <summary>
+        /// Context creation flags.
+        /// </summary>
+        public ContextFlags Flags { get; set; }
+
+        /// <summary>
+        /// The version of the API to use.
+        /// </summary>
+        public Version APIVersion { get; set; }
```

Comment:
Could simplify this to just `Version`, but don't have to.

Reply:
I'm not a huge fan of using type names as variable names, but on the other hand, the `API` in the name is sort of inferred.

Put it up to a vote on the Discord? It's not a big deal either way, but I'm not entirely sure on it.

Reply:
After a voting period on Discord, I've replaced `Version APIVersion` with `APIVersion Version`.